### ISSUE TITLE
Bruteforce work very well, but my ip get banned after hours

### ISSUE THREAD

**Issue by tryppi** [RedKatz/SocialMediaHackingToolkit#44]
i tryed it on my profile and it works, but after hours the il of my wifi get banned, probably becouse im on window?

**Comment by amanpatel6778** [comment]
You are wrong, beacuse i try this tool on my Instagram account with many wrong and one original password. This tool not find my pasword and it's showing also all password wrong

**Comment by RedKatz** [comment]
probably you try to log into your account too many times and your account or your io gets banned (obviusly)
